config.go

```diff
@@ -105,6 +108,25 @@ func (c *Config) StrictSet(key string, value interface{}, time time.Time) error
 }
 */
 
+func (c *Config) Unset(key string) error {
+	configMap := c.v.AllSettings()
+	delete(configMap, key)
+	tomlTree, err := toml.TreeFromMap(configMap)
+	if err != nil {
+		return err
+	}
+	configFile := c.v.ConfigFileUsed()
+	// Need a new viper instance to claer old settings
+	c.v = viper.New()
+	c.v.SetFs(fs)
+	c.v.SetConfigFile(configFile)
+	if err := c.v.ReadConfig(bytes.NewReader([]byte(tomlTree.String()))); err != nil {
```

Comment:
tomlTree.String()  calls tomlTree.Marshal under the hood, which returns a []byte and converts it to string, which you then turn back into a []byte. It would be better to just call Marshal directly.